packages/flutter/lib/src/material/material.dart

```diff
@@ -14,6 +14,64 @@ import 'theme.dart';
 /// Used by [InkHighlight] and [InkSplash], for example.
 typedef RectCallback = Rect Function();
 
+/// Interactive states that some of the Material widgets can take on when
+/// receiving input from the user.
+///
+/// States are defined by https://material.io/design/interaction/states.html#usage.
+///
+/// Some Material widgets track their current state in a `Set<MaterialState>`.
+///
+/// See also:
+///  * [MaterialStateColor], a color that takes a callback function that can
```

Comment:
Please update this description to account for the fact that MSC isn't required to be defined in terms of a callback.

Reply:
Done.